src/redux/functionServise.js

```diff
@@ -0,0 +1,16 @@
+export function getTotalBalance(payload) {
+  //   console.log('payload', payload);
+  const byType = payload.map(el => (el.type === true ? el.sum : null));
+  //   console.log('byType', byType);
+
+  //   const sum = byType.reduce((previousValue, current) => {
+  //     if (previousValue === 'number' && current === 'number')
+  //       console.log('previousValue', previousValue, current);
+  //     // return previousValue + current;
+  //     return current;
+  //   });
+  //   console.log('sum', sum);
+
+  //   return sum;
+  return 320;
+}
```

Comment:
I think this function is responsible for getting user balance, so rename it according to it's purpose

Reply:
renamed to"userBalace"